src/extension/csv-with-metadata-header/src/main/java/au/org/emii/geoserver/wfs/response/DataDirectory.java

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Would be good if we could leverage the existing code instead of copying it.  Can just include filters as a dependency or separate out code to be reused into a separate module.

Reply:
Good point. I didn't know I could do that...